nautobot/cloud/forms.py

```diff
@@ -275,7 +276,7 @@ class CloudServiceBulkEditForm(TagsBulkEditFormMixin, NautobotBulkEditForm):
     pk = forms.ModelMultipleChoiceField(queryset=CloudService.objects.all(), widget=forms.MultipleHiddenInput)
     description = forms.CharField(max_length=CHARFIELD_MAX_LENGTH, required=False)
     cloud_account = DynamicModelChoiceField(queryset=CloudAccount.objects.all(), required=False)
-    cloud_network = DynamicModelChoiceField(queryset=CloudNetwork.objects.all(), required=False)
+    cloud_networks = DynamicModelMultipleChoiceField(queryset=CloudNetwork.objects.all(), required=False)
```

Comment:
Does this work as intended in a bulk-edit form? I thought normally for bulk-edit of M2M fields we had an `add_foo` and `remove_foo` form field instead + special form save logic?